constellations/orion/src/repos/achieve_repo.go

```diff
@@ -105,7 +107,7 @@ func (ar *achieveRepo) SelectAllUnpublished() ([]domains.Achieve, error) {
 func (ar *achieveRepo) SelectAllGroupedByYear() ([]domains.AchieveYearGroup, error) {
 	results := make([]domains.AchieveYearGroup, 0)
 
-	stmt, err := ar.db.Prepare("SELECT * FROM achievements ORDER BY year DESC")
+	stmt, err := ar.db.Prepare("SELECT * FROM achievements ORDER BY year DESC, position ASC")
```

Comment:
This line returns the achievements in the correct order in `MySQL`, but it seems that when `rows, err := stmt.Query()` is run, this order is disregarded and the first row in `rows` just becomes the first row that is added. @ahsu1230 @mythicat554